client/components/ModalView.jsx

```diff
@@ -1,21 +1,79 @@
 import React from 'react';
 import styled from 'styled-components';
 
-const Image = styled.img`
-  height: 250px;
-  width: 250px;
-  object-fit: cover;
+
+const PictureWrapper = styled.div`
+  display:flex;
+  flex-direction: row;
+  align-items: center;
+  z-index: -100;
 `;
 
-const CurrImage = styled.img`
-  object-fit: cover;
+const NxButton = styled.button`
+  cursor: pointer;
+  z-index: 50;
+  order: 3;
+  background: transparent;
+  border: none;
+`;
+
+const PrvButton = styled.button`
+  cursor: pointer;
+  z-index: 50;
+  order: 1;
+  background: transparent;
+  border: none;
+`;
+
+const NextIcon = styled.svg`
+  flex: none;
+  height: 100%;
+  width: 100%;
+  z-index: 500;
+  position: relative;
+  margin-top: 200%;
+  transition: fill 0.25s;
+  ${NxButton}: hover & {
+    fill: rebeccapurple;
+  }
+`;
+
+const PreviousIcon = styled.svg`
+  flex: none;
   height: 100%;
   width: 100%;
+  z-index: 500;
+  position: relative;
+  margin-top: 200%;
+  transition: fill 0.25s;
+  ${PrvButton}: hover & {
+    fill: rebeccapurple;
+  }
+`;
+
+const CurrImage = styled.img`
+  order: 2;
+  z-index: 1;
+  object-fit: contain;
+  height: 100%;
+  width: 80%;
 `;
 
 const ModalView = (props) => (
```

Comment:
- good use of functional stateless component when you don't need a state 👍 
- could go without parentheses around props if you really want to stream line it :)

Reply:
adding some comments that weren't from this PR as well by adding "issues".